formatting on rabbitmq credentials

xiujiao opened the pull request.

xiujiao commented:
Will update documentation later

jhunt commented on README.md:
````diff
@@ -167,22 +167,32 @@ applications:
-- `rmq_port` - The TCP port that RabbitMQ's AMQP interface is
-  bound to and listening for inbound connections on.
+```
+- `host/hostname` - The IP address of the standalone RabbitMQ host.
````
The `teletext` formatting made me initially think that `host/hostname` was the literal key name.  In reality, `host` and `hostname` are both interchangeable -- can we update the formatting to convey this?

xiujiao replied:
fixed :)

jhunt commented on README.md:
```diff
@@ -167,22 +167,32 @@ applications:
+  },
+  "rmq_port": 5672,
+  "uri": "amqp://some-user:sekrit@10.x.x.x:5672",
+  "username": "some-user"
```
Can you match the existing formatting and line up the `:`?  It should make both the diff patch and the docs easier to read.

xiujiao replied:
yes, done

jhunt commented on README.md:
```diff
@@ -192,21 +202,37 @@ applications:
   RabbitMQ.  It contains all of the other values, in a URL / DSN
   for easier integration with libraries / SDKs.
 
+The section in the protocols block is to also provide Pivatol RabbitMQ equivalent
```
"Pivotal" is mispelled.

xiujiao replied:
whoops, fixed

jhunt commented on README.md:
```diff
@@ -192,21 +202,37 @@ applications:
 
+The section in the protocols block is to also provide Pivatol RabbitMQ equivalent
+credentials formatting to applications. This is in order that applications 
+written to work with Pivotal RabbitMQ may work without changes to the applciations.
```
Can you reword this?

xiujiao replied:
I deleted second sentence

jhunt commented on README.md:
```diff
@@ -192,21 +202,37 @@ applications:
+  },
+  "rmq_port": 5672,
+  "uri": "amqp://some-user:sekrit10.x.x.x:5672",
+  "username": "some-user"
```
As above, please match formatting.

jhunt commented on README.md:
```diff
@@ -232,6 +258,11 @@ applications:
   applications should use this list to failover on node failure.
 
+
+The section in the protocols block is to also provide Pivatol RabbitMQ equivalent
```
"Pivotal" is misspelled.

jhunt commented on jobs/rabbitmq-blacksmith-plans/templates/plans/cluster/credentials.yml:
```diff
@@ -7,3 +7,15 @@ credentials:
+      ssl: false
+      uris: (( grab credentials.uris ))
+      uri:  (( grab credentials.uris[0] ))
+      username: (( grab meta.username ))
```
Can you line up the opening `((` to match the existing style?

xiujiao replied:
done

xiujiao commented:
@jhunt Could you please take a took when you get a chance to see if I addressed all your comments? After you are good with it, would like to merge and cut a final release.
